kevm_pyk/src/kevm_pyk/__main__.py

```diff
@@ -48,9 +54,48 @@ def main():
         else:
             kevm = KEVM(args.definition_dir)
 
+            kevm.symbol_table['hashedLocation'] = lambda lang, base, offset: '#hashedLocation(' + lang + ', ' + base + ', ' + offset + ')'  # noqa
```

Comment:
Consider extracting the side effect free part of these branches into functions of their own.

Reply:
Not quite sure what you mean, maybe you can make a follow-up clean-up PR? Would rather just get this functionality merged.